extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -135,24 +135,27 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   assertSource_(src, containerSrc, sandbox = '') {
-    const url = parseUrlDeprecated(src);
+    const {element: el} = this;
```

Comment:
nit: `el` seems unnecessarily abbreviated, but it's probably a personal taste thing.

Reply:
True, fixed.